src/Card/README.md

```diff
@@ -354,7 +360,9 @@ When using horizontal variant Paragon provides additional component `Card.Body`
         </Card.Footer>
       </Card>
       <Card className="mb-4" orientation={isSmall ? "vertical" : "horizontal"}>
-        <Card.ImageCap 
+        <Card.ImageCap
+          skeletonHeight={isSmall && 140}
+          capLogoSkeletonHeight={41}
```

Comment:
If this `Card` isn't in a loading state (no `isLoading` prop), do we need the `skeletonHeight` and `capLogoSkeletonHeight` props here?